tests/test_kernels.py

```diff
@@ -700,6 +700,48 @@ def test_connectivities_key_kernel(self, adata: AnnData):
         np.testing.assert_array_equal(T_cr.A, adata.obsp[key])
 
 
+class TestVelocityKernelReadData:
+    @pytest.mark.parametrize("attr", ["layers", "obsm"])
+    @pytest.mark.parametrize("use_gene_subset", [True, False])
+    def test_read_correct_from_layers(
+        self,
+        adata: AnnData,
+        attr: str,
+        use_gene_subset: bool,
+        xkey: str = "Ms",
```

Comment:
Please remove `xkey/vkey` from the function signature and just hardcode them in the function.